### ISSUE TITLE
Save a formula or any string which starts with `"="` as a string using the cells-based interface

### ISSUE THREAD

**Issue by miron1512** [theoephraim/node-google-spreadsheet#644]
Hi @theoephraim,
There is currently no way to save a formula or any string which starts with `"="` as a string using the cells-based interface.
I think that if we explicitly use `cell.stringValue`, we already know that we want to save the value as `string`, regardless of whether it's a formula or any string which starts with `"="`. So, we don't need to throw an error here:

![image](https://github.com/theoephraim/node-google-spreadsheet/assets/10398783/2d930715-7ae5-478c-a1eb-dd5d0a2fe749)

I created a small PR to fix this: #643

**Comment by theoephraim** [comment]
yeah I think this is right. I swore I had already made it work this way, but might have gotten lost while doing some other bigger refactoring. Will get this merged in with some tests shortly.
